app.ts

```diff
@@ -1,17 +1,20 @@
 require('dotenv').config();
-const express = require('express');
+import express, { Request, Response } from 'express';
+import { swaggerSpec } from './utils/swagger';
+
 const morgan = require('morgan');
 const cors = require('cors');
 
 const app = express();
 const port = 3000;
 
 // Load Mock Modules
-require('./modules/animal/api/animal-routes')(app) // Animals
-require('./modules/chat/api/chat-routes')(app) // Chat
+require('./modules/animal/api/animal-routes')(app); // Animals
```

Comment:
why are you using require in a typescript file? Why not `import {thing} from 'thing'`

Reply:
There has been no real conversation around this. This was just the way when I was learning node. If there is a better way, please can you raise a ticket with information in it so we can look into doing that. Thanks @benhalverson 👍